Create testing-push-notifications.md (#58)
atolye15/checklist: checklists/testing-push-notifications.md

---
title: Testing Push Notification
category: Project Management
date: "2020-03-11"
tags: ['test', 'notification']
description: Push notifications are vital for mobile applications, and you need to test them carefully. This checklist will help you to cover all cases.
---

- [ ] Test the notification while app is on foreground  
Carefully test all the wordings and emojis.

- [ ] Test the notification in the different screens of the app  
Take care of the modals and try to test each screen. 

- [ ] Check the notification while app is on background  

> Notification wording might be changed while app is on background. Check the wording if it is same with related Jira issue or not.

Reviewer: ```suggestion
Notification wording might change while the app is running on background. Check the wording whether it's the same as related Jira issue or not.
```

> - [ ] Check the notification is still coming while app is closed  

Reviewer: ```suggestion
- [ ] Check to see if the user still gets the expected notification even when the app is closed  
```

> - [ ] Do all test on both IOS and Android devices 

Reviewer: ```suggestion
- [ ] Apply the tests on both iOS and Android devices  
```

> - [ ] Try to test all different notifications

Reviewer 2: Maybe we can add testing of deep linking. @ecempehlivan 

```suggestion
- [ ] Try to test all different notifications

- [ ] Test if deep linking works correctly
```
